Mac/BuildScript/build-installer.py

```diff
@@ -359,10 +359,9 @@ def library_recipes():
                   ),
           ),
           dict(
-              name="SQLite 3.42.0",
-              url="https://sqlite.org/2023/sqlite-autoconf-3420000.tar.gz",
-              checksum="0c5a92bc51cf07cae45b4a1e94653dea",
-              extra_cflags=('-Os '
+              name="SQLite 3.43.1",
+              checksum="fd32866c281539eae95cd90b5c2c34337d8808822a988211b9b009c1e788e42d",      
+	      extra_cflags=('-Os '
```

Comment:
Please revert this unrelated indentation change.

Reply:
Now fixed on my end, will push soon.